SystemInformer/tokprp.c

```diff
@@ -3580,6 +3588,100 @@ PPH_STRING PhFormatTokenSecurityAttributeValue(
         return string;
     }
 
+    // WIN://BGKD
+    if (ValueIndex == 0 &&
+        Attribute->ValueType == TOKEN_SECURITY_ATTRIBUTE_TYPE_INT64 &&
+        PhEqualStringRef(Name, &winBgkd, TRUE))
+    {
+        switch (Attribute->Values.Uint64[0])
+        {
+        case PsmActNotBackground:
+            PhInitFormatS(&format[0], L"Not background");
+            break;
+        case PsmActMixedHost:
+            PhInitFormatS(&format[0], L"Mixed host");
+            break;
+        case PsmActPureHost:
+            PhInitFormatS(&format[0], L"Pure host");
+            break;
+        case PsmActSystemHost:
+            PhInitFormatS(&format[0], L"System host");
+            break;
+        default:
+            PhInitFormatS(&format[0], L"Invalid");
```

Comment:
nit: maybe "Unknown" instead? For consistency elsewhere?

Reply:
Yeah, perhaps. The enum defines `PsmActInvalidType`, but I don't think anything is supposed to use it